contracts/sweep/SweepERC20.sol

```diff
@@ -32,6 +32,6 @@ abstract contract SweepERC20 is TokenSweep {
             tokenSweepBeneficiary(),
             amount
         );
-        require(result, "SweepERC20: transfer failed");
+        require(result, "SweepERC20: transfer failed"); //
```

Comment:
question: why the `//`?

Reply:
oh was leaving it in here as a reminder to myself -should we be using Safe Transfer here?

Reply:
Hmmm.... `safeTransfer` 🤔 

We could, but we should apply it consistently i.e. use it everywhere.

Unless you've already started using it (and already imported the lib), I'd raise an issue to apply everywhere once the contracts have been split and downsized.

Reply:
opened https://github.com/windranger-io/windranger-treasury/issues/328